packages/wrangler/src/pages/index.tsx

```diff
@@ -81,6 +82,12 @@ export const pages: BuilderCallback<unknown, unknown> = (yargs) => {
 							Publish.Options,
 							Publish.Handler
 						)
+						.command(
```

Comment:
can we make sure that `wrangler pages tail --help` shows the appropriate help message here? we're referencing this command in the docs